<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>840</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="frame" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>850</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#frame{
    
	background-color: qlineargradient(spread:pad, x1:0.467, y1:0.37, x2:1, y2:1, stop:0 rgba(0, 80, 138, 255), stop:1 rgba(255, 255, 255, 255));
	
}</string>
   </property>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>721</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20 20pt &quot;Sitka Heading Semibold&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Make your unforgettable train experience with us!</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputName">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputEmail">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>160</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="nameLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20 17pt &quot;Sitka Heading Semibold&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="emailLabel">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>110</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20 17pt &quot;Sitka Heading Semibold&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QPushButton" name="signupButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>600</y>
      <width>361</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius: 20px;
background-color: rgb(255, 255, 255);
font: 63 12pt &quot;Sitka Small Semibold&quot;;
color:rgb(0, 86, 148);
}

QPushButton:hover{
	background-color: #00508a;
	color:white;
}</string>
    </property>
    <property name="text">
     <string>Signup</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputPassword">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputNumber">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>290</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="passwordLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20 17pt &quot;Sitka Heading Semibold&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="phoneLabel">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>240</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20 17pt &quot;Sitka Heading Semibold&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Phone Number</string>
    </property>
   </widget>
   <widget class="QLabel" name="dobLabel">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>360</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20 17pt &quot;Sitka Heading Semibold&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Date of birth</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputAddress">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="addressLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20 17pt &quot;Sitka Heading Semibold&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Address</string>
    </property>
   </widget>
   <widget class="QLabel" name="roleLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>480</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20 17pt &quot;Sitka Heading Semibold&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Role</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="inputDob">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>410</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 4px;
font: 20 17pt &quot;Sitka Heading Semibold&quot;;
color: black;</string>
    </property>
    <property name="maximumDateTime">
     <datetime>
      <hour>23</hour>
      <minute>59</minute>
      <second>59</second>
      <year>2023</year>
      <month>5</month>
      <day>21</day>
     </datetime>
    </property>
    <property name="minimumDateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>1900</year>
      <month>1</month>
      <day>1</day>
     </datetime>
    </property>
    <property name="maximumTime">
     <time>
      <hour>23</hour>
      <minute>59</minute>
      <second>59</second>
     </time>
    </property>
   </widget>
   <widget class="QRadioButton" name="customerRadioButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>510</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20 10pt &quot;Sitka Heading Semibold&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Customer</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="adminRadioButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>510</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20 10pt &quot;Sitka Heading Semibold&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Admin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="returnButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>780</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius: 20px;
background-color:qlineargradient(spread:pad, x1:0.467, y1:0.37, x2:1, y2:1, stop:0 rgba(0, 80, 138, 255), stop:1 rgba(255, 255, 255, 255));
font: 63 12pt &quot;Sitka Small Semibold&quot;;
color:white;
}

QPushButton:hover{
	background-color: white;
	color: #00508a;
}</string>
    </property>
    <property name="text">
     <string>Return</string>
    </property>
   </widget>
   <widget class="QLabel" name="error">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>550</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Sitka Heading Semibold&quot;;
color:rgb(180, 0, 0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="errorMsg">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>670</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Sitka Heading Semibold&quot;;
color:rgb(180, 0, 0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="emailError">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>210</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Sitka Heading Semibold&quot;;
color:rgb(180, 0, 0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
